misc/helm-charts/operator/README.md

```diff
@@ -37,19 +37,31 @@ The following table lists the configurable parameters of the Materialize operato
 
 | Parameter | Description | Default |
 |-----------|-------------|---------|
+| `cockroachdb.caCert` |  | ``"-----BEGIN CERTIFICATE-----\nTODO\n-----END CERTIFICATE-----\n"`` |
+| `cockroachdb.endpoint` |  | ``"cockroachdb-public.cockroachdb.svc.cluster.local"`` |
+| `cockroachdb.password` |  | ``"password"`` |
+| `cockroachdb.port` |  | ``26257`` |
+| `cockroachdb.username` |  | ``"environment-controller"`` |
```

Comment:
this is one spot where I might start referencing it as the metadata db (though I also think it might go away soon, if we're able to move access entirely to the Materialize CR)